### DISCUSSION TITLE
Gator Grader Issue.

### DISCUSSION THREAD

**Discussion by cheng-cs** [ProactiveProgrammers/www.proactiveprogrammers.com#11] [Q&A]
Hello, I am currently running into an issue in my terminal. When I try to use Gater Grader, though both commands, it outputted a giant error message of nonsense. To clarify, Gater Grader has been running for me for the last couple of years. This is an issue that poped up recently. Does anyone have any tips on how to resolve this issue?

**Comment by gkapfham** [comment]
Hello @cheng-cs thanks for reporting this issue! Could I trouble you to please share the exact command that you are running and the exact output that you see in your terminal window? Once you provide this additional information we can work together to resolve the issue.
